presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5962,12 +5962,8 @@ public void testInformationSchemaFiltering()
     @Test
     public void testSelectColumnOfNulls()
     {
-        // Currently nulls can confuse the local planner, so select some
-        assertQueryOrdered("SELECT \n" +
-                " CAST(NULL AS VARCHAR),\n" +
-                " CAST(NULL AS BIGINT)\n" +
-                "FROM ORDERS\n" +
-                " ORDER BY 1");
+        // Currently nulls can confuse the local planner, so SELECT some
```

Comment:
I don't understand the comment, but the change in it looks like unintentional. Also, since I don't understand this comment, I can't say whether the new version of this test still covers the original problem.

Reply:
I removed that change.